CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+Plugin: A new plugin hook, `rpc_command` allows a plugin to take over `lightningd` for any RPC command.
```

Comment:
Hey, can we use the new commit line, in anticipation of Christian's Changelog tool being merged?

i.e. in commit msg, put
Changelog-added: Plugin: A new plugin hook, `rpc_command` allows a plugin to take over `lightningd` for any RPC command.

Reply:
Will try !